widgetssdk/src/main/java/com/glia/widgets/GliaWidgets.java

```diff
@@ -163,6 +165,22 @@ private static void setAuthorization(
         }
     }
 
+    private static void listenCallVisualizerEvents() {
+        Dependencies.glia().getCallVisualizer().on(Omnibrowse.Events.ENGAGEMENT_REQUEST, engagementRequest -> {
+            Consumer<GliaException> onResult = error -> {
+                if (error != null) {
+                    Log.e(TAG, "Error during accepting engagement request, reason" + error.getMessage());
```

Comment:
Shouldn't it be the `Logger` instead of `Log`?

Reply:
Fixed 2fe4619f2de6e8e72589ce96d2fa876595989249